src/js/index.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { render } from "react-dom";
+import { Provider } from "react-redux";
+import store from "./store/index";
+import App from "./components/App";
```

Comment:
you should replace the link './components/App' into 'components/App' with a litle bit configuration in webpack (resolve)
In the next projects if you use eslint you need also use eslint-import-resolver-webpack in order to help eslint to understand your configuration about resolved path of webpack